src/cmd/linuxkit/run_qemu.go

```diff
@@ -398,6 +404,27 @@ func runQemuLocal(config QemuConfig) error {
 		return fmt.Errorf("Detached mode is only supported when running in a container, not locally")
 	}
 
+	if len(config.VirtiofsShares) > 0 {
+		args = append(args, "-object", "memory-backend-memfd,id=mem,size="+config.Memory+"M,share=on", "-numa", "node,memdev=mem")
+	}
+	for index, source := range config.VirtiofsShares {
+		socket := fmt.Sprintf("%s%d", "/tmp/virtiofs", index)
+
+		cmd := exec.Command("/usr/lib/qemu/virtiofsd", "--socket-path="+socket,
+			"--socket-group="+"docker",
```

Comment:
I'm not sure what this does and the documentation from https://gitlab.com/virtio-fs/virtiofsd does not shed any light

Reply:
Will drop it, it's a remnant of trying to as non root